softlayer/vm/network.go

```diff
@@ -27,6 +27,15 @@ func (ns Networks) First() Network {
 	return Network{}
 }
 
+func (n Network) IsDefaultGateway() bool {
```

Comment:
This might be better called `HasDefaultGateway()` since you are searching to see if a gateway exists in the default networks?